src/main/java/com/avispl/symphony/dal/logitech/collabos/LogitechCollabOsCommunicator.java

```diff
@@ -68,6 +73,49 @@ public class LogitechCollabOsCommunicator extends RestCommunicator implements Mo
 	 */
 	private String token;
 
+	/**
+	 * save time get token
+	 */
+	private Long tokenExpire;
+
+	/**
+	 * time the token expires
+	 */
+	private Long expiresIn = 12 * 3600L * 1000;
+
+	/**
+	 * failed monitor
+	 */
+	int failedMonitor = 0;
```

Comment:
does this need to be a default access, or private?

Reply:
> does this need to be a default access, or private?

Sure, I miss it :D